fs.go

```diff
@@ -164,6 +164,7 @@ type File interface {
 	// Name returns the name of the file as presented to Open.
 	Name() string
 	io.Writer
+	io.WriterAt
```

Comment:
This isn't backwards compatible. We are better off just adding the `WriteAt` func now, and then introducing the interface change on `v6`.

```suggestion
	// TODO: Add io.WriterAt for v6  
	// io.WriterAt
```

Users will have to do type assertion to check their specific version of File supports WriterAt, before using it.